packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -88,5 +95,64 @@ test.describe('`Pie Modal is closed`', () => {
             expect(messages).toHaveLength(1);
         });
     });
-});
 
+    test('should return focus to specified element', async ({ mount, page }) => {
```

Comment:
I think these would be good to have a wrapping describe.

`test.describe('returnFocusAfterCloseSelector: when specified element is passed in')`
    `test('should return focus to that element')`

Reply:
Good shout, updated